nosql/1.0/_index.md

```diff
@@ -46,12 +100,13 @@ template.delete(Car.class, 1L);
 
 
 * [Jakarta NoSQL Release Plan](https://projects.eclipse.org/projects/ee4j.nosql/governance)
```

Comment:
This isn't quite the right link
```suggestion
* [Jakarta NoSQL Release Plan](https://projects.eclipse.org/projects/ee4j.nosql/releases/1.0)
```

Reply:
@otaviojava 
Please address @Pandrex247 comments.

Reply:
Done!